IssueTracker/IssueTracker.FileSystem/Repository/IRepository/IMetaDataProvider.cs

```diff
@@ -2,10 +2,10 @@
 
 namespace IssueTracker.FileSystem;
 
-public interface IMetaDataProvider
+internal interface IMetaDataProvider
 {
     IEnumerable<MetaDataResponse> GetAll();
-    Task CreateAsync(MetaDataRequest entity);
+    Task CreateAsync(FileModel entity); //I change MetadataRequest in FileModel
```

Comment:
Is this comment intended to be merged into the main branch?

Reply:
no, this comment will be deleted